lib/systems/include/MainChargeSystem.h

```diff
@@ -3,27 +3,20 @@
 
 
 #include "CCUEthernetInterface.h"
+#include "CCUParams.h"
+
 
 
 class MainChargeSystem {
```

Comment:
Does this necessarily need to be a class? It's just one function and it's not stateful